### PR TITLE
Problem solved  Feb.8

### PR DESCRIPTION
**Pull request by ryoosukesaito**
There were  problems which I solved.

### CODE REVIEW

File: 1502.can-make-arithmetic-progression-from-sequence.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * https://leetcode.com/problems/can-make-arithmetic-progression-from-sequence/description/
+ */
+
+/**
+ * @param {number[]} arr
+ * @return {boolean}
+ */
+
+var canMakeArithmeticProgression = function (arr) {
+  const sortedNum = arr.sort((a, b) => a - b);
```
Comment (yuya-minamide) [inline 1502.can-make-arithmetic-progression-from-sequence.js:11]: I think this name should be plural

File: 1502.can-make-arithmetic-progression-from-sequence.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  if (arr.length < 3) {
+    return true;
+  }
+
+  const numDiff = sortedNum[1] - sortedNum[0];
```
Comment (yuya-minamide) [inline 1502.can-make-arithmetic-progression-from-sequence.js:17]: I'd say difNums is better.
It's just my opinion though.

File: 415.add-strings.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * https://leetcode.com/problems/add-strings/description/
+ */
+
+/**
+ * @param {string} num1
+ * @param {string} num2
+ * @return {string}
+ */
+var addStrings = function (num1, num2) {
+  const amount = BigInt(num1, 10) + BigInt(num2, 10);
```
Comment (yuya-minamide) [inline 415.add-strings.js:11]: Is there any reason you put 10 in the second argument of BigInt()?

**Review by yuya-minamide — APPROVED** [review]

File: 1502.can-make-arithmetic-progression-from-sequence.js (hunk 1 above)
Comment (ryoosukesaito) [reply 1502.can-make-arithmetic-progression-from-sequence.js:11]: Im gonna change these things thx

File: 415.add-strings.js (hunk 3 above)
Comment (ryoosukesaito) [reply 415.add-strings.js:11]: i just used parseInt in here  before changing operator then, 10 meant decimal.
if its not necessary, I'm gonna delete it.

**Pull request merged**